src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/RuntimeFeatureTests.cs

```diff
@@ -30,7 +30,7 @@ public static void DynamicCode()
         }
 
         [Fact]
-        [SkipOnMono]
+        [SkipOnMono("Crashes with IsDynamicCodeCompiled interpreter set to false")]
```

Comment:
It doesn't actually "crash", right?  The property just returns false in some cases.

Reply:
True. I just read the description and submitted the fix as quickly as possible :smile:

Reply:
Fixed.